test/simple_mc_integration.cpp

```diff
@@ -66,33 +66,41 @@ void  call_test_simple_mc_integration_over_rectangles() {
 	typedef HPolytope<Point> HPOLYTOPE;
 	typedef VPolytope<Point> VPOLYTOPE;
 	typedef boost::mt19937 RNGType;
-	typedef BoostRandomNumberGenerator<RNGType, NT> RandomNumberGenerator;
-	
-	NT integration_value;
+	typedef BoostRandomNumberGenerator<RNGType, NT, 0> RandomNumberGenerator;
+
 	std::cout << "\nTESTS FOR SIMPLE MC INTEGRATION OVER RECTANGLES USING UNIFORM RANDOM WALKS\n";
 
+    NT integration_value;
```

Comment:
indentation looks off here. Could you please use 4 space indentation in the whole function (even better in the whole file)?